manifests/init.pp

```diff
@@ -229,6 +234,7 @@
   Boolean          $use_netgroups             = false,
   Boolean          $use_openshift             = false,
   Boolean          $sssd                      = simplib::lookup('simp_options::sssd', { 'default_value' => false}),
+  Boolean          $enable_separator          = true,
```

Comment:
Yes, we want it set to  true by default users are able to use spaces in groups and names.  I explained earlier how we found that the problem was with our adding spaces and Trevor did not want to remove the code Dylan put in to make the list separator optional, so I defaulted it to true and removed the spaces from our rule add.... then I updated all the tests.